_sass/module/_hero.scss

```diff
@@ -9,39 +9,37 @@ $hero-breakpoint: $grid--medium;
 }
 
 .hero__attribution {
+  position: absolute;
   background-color: $attribution-background-color;
-  bottom: 0;
-  display: none;
   font-size: $small-font-size;
+  text-align: center;
   padding: 0 $spacing-base;
-  position: absolute;
-  right: 0;
+  width: auto;
+  bottom: 0px;
+  right: 0px;
 
   @include media($hero-breakpoint) {
     display: block;
   }
 }
-
 .hero--photo {
-  @include media($hero-breakpoint) {
-    background-image: url('./images/open-disclosure-splash.png');
-    background-position: bottom;
-    background-size: cover;
+  background-image: url('./images/open-disclosure-splash.png');
+  background-position: bottom;
+  background-size: cover;
 
+  @include media($hero-breakpoint) {
     // When hero includes a background image, increase legibility
```

Comment:
Based on this comment, I think we want to remove the `media` mixin here, because any `.hero--photo` will include the background image and will need increased legibility.